### PR TITLE
chore(docs): create missing markdown files for cdk items

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: src/cdk/keycodes/keycodes.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  selector: '[count-arrows]'
+  host: {
+    (keypress): 'handleKeyPress($event)'
+  }
+})
+export class ArrowCounterDirective {
+  arrowPressCount = 0;
+
+  handleKeyPress(event: KeyboardEvent) {
+    if (event.codeCode in [UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW]) {
```
Comment (crisbeto) [inline src/cdk/keycodes/keycodes.md]: 1. `event.codeCode` -> `event.keyCode`.
2.  This check will look for a property that corresponds to `event.keyCode` in the `Array` prototype. The correct way to check it is `[UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW].includes(event.keyCode)`.

File: src/cdk/overlay/overlay.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+### Overlay
+
+A component used to manage overlays.
```
Comment (crisbeto) [inline src/cdk/overlay/overlay.md]: `Overlay` is a service, not a component.

File: src/cdk/rxjs/rxjs.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+### RxJS Usage
+When dealing with RxJS operators, it is best to import the operator functions directly (e.g.
```
Comment (crisbeto) [inline src/cdk/rxjs/rxjs.md]: This should mention that avoiding the patch operators is only recommended when writing a library.

File: src/cdk/rxjs/rxjs.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+```ts
+// Before
+someObservable.filter(...).map(...).do(...);
+
+// After
+RxChain.from(someObservable).call(filter, ...).call(map, ...).subscribe(...);
+```
+
+#### Available Operators
+Many operators are provided in via the `RxChain`, these operators include:
```
Comment (crisbeto) [inline src/cdk/rxjs/rxjs.md]: I don't think that we should include the list operators here, because it means that we'll have to maintain it.

Comment (josephperrott) [reply src/cdk/rxjs/rxjs.md]: Yeah, you're right.  I went back and forth on it before I put it in.

**Pull request merged**